src/appendices/spherical-harmonics.md

```diff
@@ -0,0 +1,106 @@
+# Spherical Harmonics
+
+Spherical Harmonics (SH) are continuous functions defined on the surface of the sphere.
+They may be used to encode directionally dependent information;
+for instance, the distribution of white matter fibre orientations within an image voxel.
+This page provides the requisite information on the different basis sets that may be used,
+and how coefficients of such functions may be encoded within a data file.
+
+## SH functions
+
+![SH functions](https://latex.codecogs.com/gif.latex?Y_l^m(\theta,\phi)&space;=&space;\sqrt{\frac{(2l&plus;1)}{4\pi}\frac{(l-m)!}{(l&plus;m)!}}&space;P_l^m(\cos&space;\theta)&space;e^{im\phi}")
+
+for integer *order* *l*, *phase* *m*, associated Legendre polynomials *P*.
```

Comment:
Consider "phase factor" instead of "phase"? See https://github.com/dipy/dipy/pull/3086#issuecomment-1977956755 and preceding discussing there.

Reply:
Lately I've tried to consistently use "degree" rather than "order" for *l*. Wikipedia uses "degree" and "order". "Phase factor" is at risk of erroneous conflation with specifically the Condon-Shortley phase factor. So I wonder if adopting "degree" and "order" here would be better?
(I'd likely modify the *MRtrix3* online documentation accordingly)

Reply:
I would really prefer to use "order" here (rather than "degree") for the reasons mentioned in that PR discussion in DIPY (i.e., to be consistent with the dMRI literature). But I agree that "phase" is fine, even though it implies something continuous, it is easy to understand what it means.

Reply:
@jdtournier any strong opinions? I have some limited sense that we've tried to transition to "degree" for *l*, but our own SH docs page conflicts, other docs self-conflict, and I'm not able to find any discussions online.

PS. This terminology is not just for the appendix, there's also a metadata field specifying maximal *l*. Could call it `"SphericalHarmonicLmax"` or the like if there's too much contention.